package.json

```diff
@@ -150,9 +146,9 @@
     "puppeteer": "^3.3.0",
     "rimraf": "^3.0.2",
     "rosie": "^2.0.1",
-    "ts-node": "^8.10.2",
-    "tslint": "^6.1.3",
-    "typescript": "~3.8.3",
+    "ts-node": "~8.3.0",
+    "tslint": "~6.1.0",
```

Comment:
Do not downgrade tslint.

Reply:
It was set by Angular 11 setup, I'll change it.

Reply:
Done